host_core/native/hostcore_wasmcloud_native/src/par.rs

```diff
@@ -39,6 +39,8 @@ pub(crate) fn extract_target_bytes(par: &ProviderArchive) -> Result<Vec<u8>, Err
     }
 }
 
+// TODO: this should respect a host ID in the future so that cached data
```

Comment:
This feels like a blocker for release?

Reply:
this is only an issue if someone is attempting to run 2 different lattices within the same host, which is not currently supported with current tooling or control interface. It'll be a blocker for when we make that functionality a first-class citizen.

Reply:
Should we file an issue to track the remaining work for first-class support of multiple lattices?

Reply:
https://github.com/wasmCloud/wasmcloud-otp/issues/505 yep.